website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -43,6 +43,10 @@ WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE T
 `0.33.0`+ _only_ supports Karpenter v1beta1 APIs and will not work with existing Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `0.36.0`+ without first [upgrading to `0.32.x`]({{<ref "#upgrading-to-0320" >}}). This version supports both the alpha and beta APIs, allowing you to migrate all of your existing APIs to beta APIs without experiencing downtime.
 {{% /alert %}}
 
+{{% alert title="Warning" color="warning" %}}
+   v0.36.x introduces update to the drift that make rollback a little restrictive, If you are coming from =>v0.36.x, note that v0.32.9+, v0.33.4+, v0.34.5+, v0.35.4+ are the patch versions that support rollback. If Karpenter is rolled back to an older patch version, Karpenter can potentially drift all the nodes in the cluster. 
```

Comment:
Some minor wording suggestions
```suggestion
   v0.36.x introduces update to drift that restricts rollback. When rolling back from v0.36.0+, note that v0.32.9+, v0.33.4+, v0.34.5+, v0.35.4+ are the patch versions that support rollback. If Karpenter is rolled back to an older patch version, Karpenter can potentially drift all the nodes in the cluster. 
```

Reply:
Leaving as >= v0.36.x. Some customer have told us that the `.x+` messages to them that they should stay within the patch version